packages/chakra-ui/src/ArrayFieldTemplate/ArrayFieldTemplate.tsx

```diff
@@ -0,0 +1,271 @@
+              style={btnStyle}
+              isDisabled={disabled || readonly || !hasMoveDown}
+              onClick={onReorderClick(index, index + 1)}
+            />
+          )}
+
+          {hasRemove && (
+            <IconButton
+              p={2}
+              mt={2}
+              minHeight="35px"
+              minWidth="35px"
+              iconMap="remove"
+              tabIndex={-1}
+              style={btnStyle}
+              isDisabled={disabled || readonly}
+              onClick={onDropIndexClick(index)}
+            />
+          )}
+        </Flex>
+      )}
+    </Flex>
+  );
+};
+
+const DefaultFixedArrayFieldTemplate = ({
+  className,
+  idSchema,
+  TitleField,
+  title,
+  required,
+  schema,
+  uiSchema,
+  items,
+  canAdd,
+  onAddClick,
+  disabled,
+  readonly,
+}: ArrayFieldTemplateProps) => {
+  return (
+    <fieldset className={className}>
+      <ArrayFieldTitle
+        key={`array-field-title-${idSchema.$id}`}
+        TitleField={TitleField}
+        idSchema={idSchema}
+        title={uiSchema["ui:title"] || title}
+        required={required}
+      />
+
+      {(uiSchema["ui:description"] || schema.description) && (
+        <div
+          className="field-description"
+          key={`field-description-${idSchema.$id}`}
+        >
+          {uiSchema["ui:description"] || schema.description}
+        </div>
+      )}
+
+      <div
+        className="row array-item-list"
+        key={`array-item-list-${idSchema.$id}`}
+      >
+        {items && items.map(DefaultArrayItem as any)}
+      </div>
+
+      {canAdd && (
+        <AddButton
+          className="array-item-add"
+          onClick={onAddClick}
+          disabled={disabled || readonly}
+        />
+      )}
+    </fieldset>
+  );
+};
+
+const DefaultNormalArrayFieldTemplate = ({
+  idSchema,
+  TitleField,
+  DescriptionField,
+  uiSchema,
+  schema,
+  title,
+  required,
+  items,
+  canAdd,
+  onAddClick,
+  disabled,
+  readonly,
+}: ArrayFieldTemplateProps) => {
+  return (
+    <Box py={2}>
+      <ArrayFieldTitle
+        key={`array-field-title-${idSchema.$id}`}
+        TitleField={TitleField}
+        idSchema={idSchema}
+        title={uiSchema["ui:title"] || title}
+        required={required}
+      />
+
+      {(uiSchema["ui:description"] || schema.description) && (
+        <ArrayFieldDescription
+          key={`array-field-description-${idSchema.$id}`}
+          DescriptionField={DescriptionField}
+          idSchema={idSchema}
+          description={uiSchema["ui:description"] || schema.description}
+        />
+      )}
+
+      <Flex
+        key={`array-item-list-${idSchema.$id}`}
+        direction="column"
+        justify="center"
+      >
+        {items && items.map((p) => DefaultArrayItem(p as any))}
+
+        {canAdd && (
+          <Flex justify="flex-end">
+            <Box mt={2}>
+              <AddButton onClick={onAddClick} isDisabled={disabled || readonly}>
+                Add More
```

Comment:
Let's either just do "Add Item", for consistency with other themes, or just do a "+".